jupyter_server/gateway/managers.py

```diff
@@ -18,16 +18,22 @@
 from jupyter_client.manager import AsyncKernelManager
 from jupyter_client.managerabc import KernelManagerABC
 from jupyter_core.utils import ensure_async
+from jupyter_events import EventLogger
 from tornado import web
 from tornado.escape import json_decode, json_encode, url_escape, utf8
 from traitlets import DottedObjectName, Instance, Type, default
 
+from .. import DEFAULT_EVENTS_SCHEMA_PATH, JUPYTER_SERVER_EVENTS_URI
 from .._tz import UTC
 from ..services.kernels.kernelmanager import AsyncMappingKernelManager
 from ..services.sessions.sessionmanager import SessionManager
 from ..utils import url_path_join
 from .gateway_client import GatewayClient, gateway_request
 
+ACTION_KEY = "action"
+KERNEL_ID_KEY = "kernel_id"
+MESSAGE_KEY = "message"
```

Comment:
This will need to match the schema property - which is currently `msg` but I've asked that it be expanded to `message`. - heads up.

Reply:
See this comment https://github.com/jupyter-server/jupyter_server/pull/1252#discussion_r1160785972

Reply:
I changed it to `msg` to be in sync with kernel_actions for now.